data_steward/common.py

```diff
@@ -52,8 +50,7 @@
 # Clinical tables which do not have a corresponding mapping table.
```

Comment:
I think this comment should be 'Clinical tables which have a corresponding mapping tables.'
I just noticed it.